tutorials/cp-starter-ibpm-employeeonboarding-1-setup/cp-starter-ibpm-employeeonboarding-1-setup.md

```diff
@@ -51,13 +51,15 @@ You will use the **Boosters** to automatically set up the workflow, business rul
 
 1. From your global account page, choose the **Boosters** from left-hand navigation. Among the available options, click **Start** of **Set up account for Workflow Management**.
 
-    !![Start Booster](startrecipe_2.png)
+    !![Start Booster](startrecipe_21.png)
+
+    > If you have multiple sub-accounts, then you will get a wizard to select the sub-account where you want to setup SAP Workflow Management capabilities. In the wizard, select the Subaccount, Org and Space. Click Finish to start the Booster.
```

Comment:
Please fix grammar. It should be "...where you want to set up...".

Reply:
Fixed this in all the places